config/routes.rb

```diff
@@ -3,10 +3,11 @@
   root "locations#index"
 
   resources :locations do
-    resources :reviews, only: [:index, :create, :new]
+    resources :reviews, only: [:index, :create, :new, :edit, :update]
```

Comment:
Not sure that this should be `only: ...` anymore. Consider changing to `except: [:show]`